core/templates/dev/head/css/oppia.css

```diff
@@ -680,10 +680,17 @@ textarea {
   height: 198px;
 }
 
+.oppia-footer-margin{
+}
+
 @media (max-width: 768px) {
   .oppia-footer-padding {
     height: 464px;
   }
+
+  .oppia-footer-margin {
```

Comment:
I will give it a try because I didn't try what you are saying.
I will give it a try and let you know here :smile: 
Thanks! :+1: